src/Sql/InsertIgnore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Db\Sql;
+
+class InsertIgnore extends Insert
+{
+    /**
+     * @var array Specification array
+     */
+    protected $specifications = [
+        self::SPECIFICATION_INSERT => 'INSERT IGNORE INTO %1$s (%2$s) VALUES (%3$s)',
```

Comment:
Are all supported drivers capable of evaluating `INSERT IGNORE`?

Reply:
probably MySQL only, that's why I didn't add `insertIgnore()` function to `Sql` and `AbstractTableGateway`

Reply:
Hmm, can we please survey that before introducing it then?